### PR TITLE
[CHORE] Rename coursedata to content

### LINKED ISSUE #1931: Rename coursedata folder
The name `coursedata` stems from way back but we don't use the word course anywhere, so maybe `content` is better. Making an issue because it might involve some renaming in the codebase too and some thorough manual testing of all languages.

### PR DESCRIPTION
**Pull request by ThomasKluiters**
**Please fill out this template by replacing all content between < >**

< Provide a one line summary of your changes in the Title of the PR >

**Description**

< Changes in detail > Use present tense without a subject to describe your PR, for example: "Adds translations of levels 1 to 12 to Polish"

Fixes #1931

< Always link the number of the issue or of the discussion that your pr concerns. >
Notable exception: adding content may be done without an accompanying issue 

**How to test**

< If this is a UI change, describe how to run your code to see it in action. See this https://github.com/Felienne/hedy/pull/880#issue-1016304308 for an example >
< If this is a language change, add a few tests showing the difference >

**Checklist**
Done? Check if you have it all in place using this list*
  
- [ ] Describes changes in the format above (present tense)
- [ ] Links to an existing issue or discussion 
- [ ] Has a "How to test" section

* If you're unsure about any of these, don't hesitate to ask. We're here to help!

### CODE REVIEW

**Comment by Felienne** [comment]
Ah thanks for picking up this chore @ThomasKluiters! It seems though that all Arabic programs are now failing, I think because by renaming somehow the rtl-ness of the code snippets got removed. I will try to see if I can get that resolved!

File: tests/test_z_defaults.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def collect_snippets(path):
     return Hedy_snippets
 
 
-Hedy_snippets = [(s.name, s) for s in collect_snippets(path='../coursedata/level-defaults')]
+Hedy_snippets = [(s.name, s) for s in collect_snippets(path=utils.construct_content_path('level-defaults'))]
```
Comment (Felienne) [inline tests/test_z_defaults.py:53]: Ah this is the offender "destroying" the Arabic snippets, we rely on the path name to find the lang. Will fix!

**Review by Felienne — APPROVED** [review]
Ah such a small change, so much work! Thanks @ThomasKluiters !!!

**Review by Felienne — CHANGES_REQUESTED** [review]
Ok the grammars and everything do work but there is an issue in app.py causing the e2e tests to fail. This is what I see on the site:

image

I am not sure what change might have caused this, maybe you can take a look yourself @ThomasKluiters? There are so many changes it is hard to see!

**Comment by ThomasKluiters** [comment]
@Felienne Will do! I thought I had marked this guy as draft as things didn't appear to be working too well :)

**Comment by Felienne** [comment]
> @Felienne Will do! I thought I had marked this guy as draft as things didn't appear to be working too well :)

Yeah I did that, I was feeling "overmoedig" and I thought I could fix it but guess not haha!

**Comment by ThomasKluiters** [comment]
> > @Felienne Will do! I thought I had marked this guy as draft as things didn't appear to be working too well :)
> 
> Yeah I did that, I was feeling "overmoedig" and I thought I could fix it but guess not haha!

Ah, "overmoedingheid" gets us all! I think the last fix should do it, but I'll test a bit manually

**Review by Felienne — APPROVED** [review]

**Comment by Felienne** [comment]
> Ah, "overmoedingheid" gets us all! I think the last fix should do it, but I'll test a bit manually

True, but together we will get there!

**Pull request merged**